src/bot/constants.py

```diff
@@ -52,7 +52,7 @@ class _DragonflyAuthentication(EnvConfig, env_prefix="auth0_"):
 
 
 class _DragonflyConfig(EnvConfig, env_prefix="dragonfly_"):
-    """Dragonfly Cog Configuration"""
+    """Dragonfly Cog Configuration."""
 
     alerts_channel_id: int = 1121462652342910986
     logs_channel_id: int = 1121462677131251752
```

Comment:
Might make more sense to move these under `_Channels`. It's references would have to be modified too.

Reply:
Constants will be cleaned up in #179